day_7/projects/fizzbuzz.rb

```diff
@@ -0,0 +1,64 @@
+# This is my solution for "FizzBuzz" when given the range 1 to 100:
+number = 1
+number.upto(100) do |num|
+  if num % 3 == 0 && num % 5 == 0 #Have to lead with this conditional otherwise "FizzBuzz" will not print to the console for 15
+    puts "FizzBuzz"
+  elsif num % 3 == 0
+    puts "Fizz"
+  elsif num % 5 == 0
+    puts "Buzz"
+  else
+    puts num
+  end
+end
+
+# *** Bonus ***
+
+#Solution one passes in any value for i to number method and runs "FizzBuzz" accordingly:
+def number(max)
+  1.upto(max) do |num|
+    str = "" #Looking at the problem from a different frame of reference and focusing on the output first as opposed to the input
+    str += "Fizz" if num % 3 == 0
+    str += "Buzz" if num % 5 == 0
+    str = num if str.empty?
+    puts str
+  end
+end
+
+number(150)
```

Comment:
Nice work using iteration and thinking through a different way of solving the same problem. It's a bit odd to see so many related individual if statements.